coordinator/core/storewrapper.go

```diff
@@ -198,25 +204,38 @@ func (s storeWrapper) putState(currState state) error {
 }
 
 // getUser returns user information from store
-// will be changed in the future to return permissions etc. instead of just certificate
-func (s storeWrapper) getUser(userName string) (*marblerunUser, error) {
+func (s storeWrapper) getUser(userName string) (*user.MarblerunUser, error) {
 	request := strings.Join([]string{requestUser, userName}, ":")
-	rawCert, err := s.store.Get(request)
+	rawUserData, err := s.store.Get(request)
 	if err != nil {
 		return nil, err
 	}
-
-	userCert, err := x509.ParseCertificate(rawCert)
+	var userData sealedUser
+	if err := json.Unmarshal(rawUserData, &userData); err != nil {
+		return nil, err
+	}
+	userCert, err := x509.ParseCertificate(userData.Certificate)
 	if err != nil {
 		return nil, err
 	}
 
-	return &marblerunUser{name: userName, certificate: userCert}, nil
+	loadedUser := user.NewMarblerunUser(userName, userCert)
+	for _, savedPerm := range userData.Permissions {
+		loadedUser.Assign(savedPerm)
+	}
+	return loadedUser, nil
 }
 
 // putUser saves user information to store
-// will be changed in the future to set permissions etc. instead of just certificate
-func (s storeWrapper) putUser(user *marblerunUser) error {
-	request := strings.Join([]string{requestUser, user.name}, ":")
-	return s.store.Put(request, user.certificate.Raw)
+func (s storeWrapper) putUser(newUser *user.MarblerunUser) error {
+	request := strings.Join([]string{requestUser, newUser.Name()}, ":")
+	userData := &sealedUser{
```

Comment:
MarblerunUser should implement the json.Marshaler interface like we did here:
https://github.com/edgelesssys/marblerun/blob/1bf8e78988700886e453ff07f773af1d88068189/coordinator/manifest/manifest.go#L169-L197